web/html/links-ii/image-mailto.md

```diff
@@ -53,12 +92,14 @@ Email me to sign up.
 
 Create a text link that opens the user's default email program.
 
-`<??? ???="??? ???">Email Me<???>`
+```html
+<??? ???="??? ???">Email Me<???>
+```
```

Comment:
formatting it like this will ensure questions gaps don't get pushed off-screen 

also, `mailto:` and `heya@enkipro.com` shouldn't have a space between them. I've suggested it like this `??????` but we should test this quickly in spider before merging (i.e. make a new workout/insight with the pq having two question gaps `??????` together)

```suggestion
```html
<??? ???="??????">
  Email Me
<???>
```
```

Reply:
If I remember correctly, two question gaps put together work fine. In the app, they are shown with a spacing smaller than a space between them. I know there are a few insights that already have them, I will check them right now to see if they work fine

Reply:
They work, there is no spacing but there is a clear visible cue that shows they are separate boxes. I am sending it to you on slack